client/src/components/playlist-duration/playlist-duration.component.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import './playlist-duration.styles.css';
+
+const PlaylistDuration = () => {
+  const likedSongsList = useSelector(
+    ({ likedSongs: { likedSongs } }) => likedSongs,
+  );
+
+  const totalSeconds = likedSongsList.reduce(
```

Comment:
Might want to create a util file to do this logic if you wanna cut down on the code inside this component.